addons/sale_timesheet/tests/test_sale_timesheet.py

```diff
@@ -969,6 +970,36 @@ def test_onchange_uom_service_product(self):
                 product_form.detailed_type = 'consu'
                 self.assertEqual(product_form.uom_id.id, uom_kg.id)
 
+    def test_non_consolidated_billing_service_timesheet(self):
+        sale_orders = self.env['sale.order'].create([{
+            'partner_id': self.partner_a.id,
+            'order_line': [Command.create({
+                'product_id': self.product_delivery_timesheet2.id,
+            })],
+        }, {
+            'partner_id': self.partner_a.id,
+            'order_line': [Command.create({
+                'product_id': self.product_delivery_timesheet2.id,
+            })],
+        }])
+        sale_orders.action_confirm()
+
+        self.env['account.analytic.line'].create([{
+            'name': 'Timesheet',
+            'task_id': task.id,
+            'project_id': task.project_id.id,
+            'unit_amount': 2,
+            'employee_id': self.employee_user.id,
+        } for task in sale_orders.tasks_ids])
+
+        advance_payment = self.env['sale.advance.payment.inv'].with_context(active_ids=sale_orders.ids).create({
+            'consolidated_billing': False,
+        })
+
+        invoices = advance_payment._create_invoices(sale_orders)
+
+        self.assertEqual(len(invoices), 2)
+
```

Comment:
oh, almost forgot, this wasn't the case beforehand, but there's supposed to empty newlines before a class defenition